Library/Formula/binwalk.rb

```diff
@@ -68,50 +53,14 @@ def install
       end
     end
 
-    if build.head?
-      ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
-      system "python", *Language::Python.setup_install_args(libexec)
-      bin.install Dir["#{libexec}/bin/*"]
-      bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
-    else
-      system "./configure", "--prefix=#{prefix}", "--disable-bundles"
-      system "make", "install"
-    end
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
 
   test do
     touch "binwalk.test"
-    system "#{bin}/binwalk", "binwalk.test"
+    system "#{libexec}/bin/binwalk", "binwalk.test"
```

Comment:
That's a real issue that'll need to be fixed as that's what users will be using.